qiskit_experiments/stored_data/utils.py

```diff
@@ -0,0 +1,219 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Experiment utility functions."""
+
+import logging
+from typing import Callable, Tuple, Dict, Any, Union
+import io
+from datetime import datetime, timezone
+import threading
+from collections import OrderedDict
+from abc import ABC, abstractmethod
+import traceback
+import pkg_resources
+
+import dateutil.parser
+from dateutil import tz
+
+from qiskit.version import __version__ as terra_version
+
+from .exceptions import ExperimentEntryNotFound, ExperimentEntryExists, ExperimentError
+
+LOG = logging.getLogger(__name__)
+
+
+def qiskit_version():
+    """Return the Qiskit version."""
+    try:
+        return pkg_resources.get_distribution("qiskit").version
+    except Exception:  # pylint: disable=broad-except
+        return {"qiskit-terra": terra_version}
```

Comment:
I think we need experiment version here.

Reply:
Fixed in c98e2c6